vizro-core/mkdocs.yml

```diff
@@ -132,3 +132,11 @@ plugins:
 
 extra_css:
   - stylesheets/extra.css
+
+# Strictest settings possible, and will be elevated to ERROR when run with --strict.
+# See https://www.mkdocs.org/user-guide/configuration/#validation.
+validation:
+  omitted_files: warn
+  absolute_links: warn
+  unrecognized_links: warn
+  anchors: warn
```

Comment:
If we want to go with this configuration, we need to update `mkdocs>=1.6` as otherwise `anchors` won't be recognised as a possible argument 👍